lib/sequelize.js

```diff
@@ -1093,5 +1119,7 @@ module.exports = (function() {
     this.connectionManager.close();
   };
 
+  // Allows the promise to access cls namespaces
+  Promise.Sequelize = Sequelize;
```

Comment:
https://github.com/sequelize/sequelize/pull/2810/files#diff-abc409e9eb06bdcdf91d5a1f0fd89ca8R39
No we went with the global setting. Shouldn't really matter though.